### PR TITLE
Check if log file location is writable in cvmfs_config chksetup

### PR DESCRIPTION
**Pull request by aandvalenzuela**

### CODE REVIEW

**Comment by aandvalenzuela** [comment]
@cernvm-bot cloudtest

File: cvmfs/cvmfs_config [hunk 1]
```diff
@@ -374,6 +374,14 @@ cvmfs_chksetup() {
     num_errors=$(($num_errors+1))
   fi
 
+  # Check that if CVMFS_USYSLOG is set, the path is writable.
+  if [ x"$CVMFS_USYSLOG" != x"" ]; then
+    if runascvmfsuser "test -w $CVMFS_USYSLOG"; then
```
Comment (jblomer) [inline cvmfs/cvmfs_config]: I think we need to make one more turn here: the file may not yet exist, in which case `test -w` returns false.  So I think we should instead check if the parent directory is writable, i.e. check for `test -w $(dirname $CVMFS_USYSLOG)`

**Review by jblomer — CHANGES_REQUESTED** [review]

File: cvmfs/cvmfs_config (hunk 1 above)
Comment (aandvalenzuela) [reply cvmfs/cvmfs_config]: Oh, true! I didn't think about the file not yet existing. Thank you!

**Comment by aandvalenzuela** [comment]
@cernvm-bot cloudtest

**Comment by jblomer** [comment]
@cernvm-bot cloudtest

**Pull request merged**